### PR TITLE
Added Arline Benchmarks, QPP, qBraid-SDK, as well as the qBraid cloning button

### PR DESCRIPTION
**Pull request by rryoung98**
Included various python packages and a C++ implemented general purpose quantum computing framework.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +37,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 **C++**
 - [staq](https://github.com/softwareqinc/staq) - [Full stack quantum processing toolkit](https://arxiv.org/abs/1912.06070).
 - [XACC](https://github.com/ORNL-QCI/xacc) - [Extreme-scale programming model for quantum acceleration within high-performance computing](https://arxiv.org/abs/1710.01794).
-
+- [qpp](https://github.com/softwareQinc/qpp) - Quantum++ is a modern C++ general purpose quantum computing library, composed solely of template header files.
```
Comment (markf94) [inline README.md:40]:
```suggestion
- [qpp](https://github.com/softwareQinc/qpp) - Quantum++ is a modern C++ general purpose quantum computing library, composed solely of template header files.

```

Needs a space

File: README.md [hunk 2]
```diff
@@ -47,6 +51,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [Strawberry Fields](https://github.com/xanaduai/strawberryfields) - [Xanadu](https://www.xanadu.ai)'s software library for photonic quantum computing.
 - [Tequila](https://github.com/aspuru-guzik-group/tequila) - Extensible Quantum Information and Learning Architecture developed by Alan Aspuru-Guzik's group (UofT).
 
+
```
Comment (markf94) [inline README.md:54]:
```suggestion
```

Comment (markf94) [reply README.md:54]: remove one space here please

**Review by markf94 — COMMENTED** [review]
Thanks for the MR! Just a few small changes before we can merge it :)

**Comment by rryoung98** [comment]
@markf94  Updated with changes:

1. qBraid SDK and airline benchmarks are now in alphabetical
2. Spacing/linting suggestions have been fixed.

**Comment by rryoung98** [comment]
Fixed merge conflicts, updated the spacing between each ##

File: README.md [hunk 3]
```diff
@@ -348,18 +353,24 @@ For a curated list of learning resources please check out [desireevl's repo](htt
...
+- [qBraid](https://qbraid-qbraid.readthedocs-hosted.com/en/latest/sdk/overview.html) - Transpiles quantum circuits to and from Cirq, Qiskit, Amazon Braket, Pennylane, Pyquil and allows for execution on any backend.
 - [qonduit](https://github.com/adgt/qonduit) - Python library with visualization tools and workflows for quantum computing that utilize the best of what’s available.
 - [qprof](https://gitlab.com/qcomputing/qprof/qprof) - `gprof`-Compatible profiler for quantum programs.
 - [QRAND](https://github.com/pedrorrivero/qrand) - Multiplatform and multiprotocol quantum random number generator for arbitrary probability distributions.
 - [QuantumGraphs](https://github.com/ziofil/QuantumGraphs) - Grow and study random graphs by a continuous, randomly collapsing quantum walk.
 - [toqito](https://github.com/vprusso/toqito) - Framework to study problems pertaining to entanglement theory, nonlocal games, and other aspects of quantum information.
 
+## Quantum data
+
+- [QDataSet](https://github.com/eperrier/QDataSet) - Quantum datasets for the training and development of QML algorithms.
```
Comment (markf94) [inline README.md:372]: this already exists on line 382 - I think you accidentally duplicated the "Quantum data" section. Please remove and then we're ready to merge :rocket:

**Comment by rryoung98** [comment]
Oof sorry about that silly mistake! Fixed 😄

**Comment by markf94** [comment]
Thanks for the contribution!

**Pull request merged**